client/src/app/templates/view/template-view.component.html

```diff
@@ -0,0 +1,35 @@
+<div fxLayoutAlign="center">
+    <mat-card fxFlex="450px" fxFlex.lt-sm="95">
+        <mat-card-header>
+            <mat-card-title fxLayout="row">
+                <a mat-button routerLink="/templates"><mat-icon>arrow_back</mat-icon></a>
+                <section class="mat-typography">
+                    <h1>{{template?.name}}</h1>
+                </section>
+            </mat-card-title>
+            <div fxFlex></div>
+            <button mat-mini-fab color="primary">
+                <mat-icon>delete</mat-icon>
+            </button>
+        </mat-card-header>
+        <mat-card-content>
+            <mat-list>
+                <h3 mat-subheader>Services</h3>
+                <div *ngIf="template.services?.length !== 0; else elseBlock">
+                    <mat-list-item *ngFor="let service of template.services">
+                        <p mat-line>{{services.startAt.day | date:'EEEE'}} de
+                          {{services.startAt.hours | date:'H'}}h {{services.startAt.minutes | date:'mm'}} à
```

Comment:
Is this working ? Moment (the `date` pipe) uses a date as entry (or a UNIX timestamp) if I remember correctly

Reply:
No it doesn't work, that's why I want to convert this to a Date